ios/Podfile.lock

```diff
@@ -425,8 +425,8 @@ PODS:
     - React-Core
   - RNStaticSafeAreaInsets (2.2.0):
     - React-Core
-  - RNSVG (9.13.6):
-    - React
+  - RNSVG (13.10.0):
+    - React-Core
```

Comment:
Hmm I am seeing this in my branch as well, it seems someone updated the SVG library and forgot to update the `Podfile.lock` , we can do that in a separate PR, Please revert all changes to `Podfile.lock` in this case in your PR